CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/Item` public class names `itemContent`, `content`, and `label`
 - `sandstone/Scroller` prop `scrollbarTrackCss` to customize scroll track and thumb style
 
+### Changed
+
+- `sandstone/Scroller` focus rule to match latest UX when `focusableScrollbar` prop is `byEnter`
```

Comment:
Please put the change log in the right place.

Reply:
Sorry. Git drove me into a corner.